### ISSUE TITLE
Slackbot Jobs not running

### ISSUE THREAD

**Issue by hderp886** [warlof/slackbot#10]
Log gives error - 

```
[2018-06-11 16:00:06] local.ERROR: Command "slack:user:kick" is not defined.

Did you mean one of these?
    slack:user:policy
    slack:user:sync
    slack:user:terminator
```

**Comment by hderp886** [comment]
I believe it's to do with the seeder? I think it's creating jobs that don't exist.

**Comment by warlof** [comment]
indead, solved in 3.0.8.

Supposed to be the following :

![image](https://user-images.githubusercontent.com/648753/41299195-3d7a5640-6e63-11e8-911f-1fbb3996f3c4.png)
